src/main/java/com/team3/central/services/CategoriesService.java

```diff
@@ -39,4 +40,16 @@ public boolean existsByName(String name) {
                 .stream()
                 .anyMatch(c -> c.getName().equals(name));
     }
+    public Set<Category> getCategoriesFromIds(List<Integer> categoriesIds)
+        throws IllegalArgumentException {
+
+        Set<Category> categories = categoryRepository.findAll()
+            .stream()
+            .filter(category -> categoriesIds.contains(category.getId().intValue()))
+            .collect(Collectors.toSet());
```

Comment:
can't we add something like get  findById in CategoryRepository class ? It would probably be faster than filtering it here